logging/google/cloud/logging/_shutdown.py

```diff
@@ -0,0 +1,131 @@
+# Copyright 2017 Google Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Module for code related to shutdown logging."""
+
+import functools
+import os
+import signal
+import sys
+import traceback
+
+from google.cloud.logging._environment_vars import _APPENGINE_FLEXIBLE_ENV_VM
+from google.cloud.logging._environment_vars import _APPENGINE_FLEXIBLE_ENV_FLEX
+
+
+# Maximum size in bytes to send to Stackdriver Logging in one entry
+MAX_PAYLOAD_SIZE = 1024 * 100
```

Comment:
Renamed, and switched order.

It's more readable to me because in my head it reads as 100KB and shows people how the eventual number is built. Keeping unless you feel strongly.